fixtures/constant-with-namespace.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace HELLO {
+
+    /**
+     * Does something really cool!
+     */
+    function world() {
+
+    }
+
+    \HELLO;
+}
+
+namespace {
```

Comment:
Yes it is. It is advised to be used when dealing with multiple namespaces in one file and need to put something in global namespace. That's what I've been taught.